src/tests/tracing/eventpipe/simpleprovidervalidation/simpleprovidervalidation.cs

```diff
@@ -29,33 +22,105 @@ public static int Main()
             // Its currently not enabled in NativeAOT runs and the below issue tracks the work
```

Comment:
Does this comment mean it is intended to run only in NativeAOT? If so, we should update the project to be disabled for other test build modes?

Reply:
It was an old comment to an issue that was resolved.